libs/api-interfaces/src/lib/aggregate-views/view-models/resources/term.view-model.interface.ts

```diff
@@ -7,7 +7,7 @@ export interface ITermViewModel extends IBaseViewModel {
 
     termEnglish?: string;
 
-    audioUrl?: string;
+    audioURL?: string;
```

Comment:
Was the controller  \ query service properly constrained using `ITermViewModel`?

Reply:
no, so I changed this.